scope.go

```diff
@@ -1230,6 +1227,40 @@ func (scope *Scope) getColumnAsArray(columns []string, values ...interface{}) (r
 	return
 }
 
+func (scope *Scope) getColumnAsArrayUnique(columns []string, values ...interface{}) (results [][]interface{}) {
+	unfilteredResults := scope.getColumnAsArray(columns, values...)
+
+	rootMap := map[interface{}]interface{}{}
```

Comment:
How come this is declared here instead of simply declaring `currentMap := map[interface{}]interface{}{}`?